tc-server/src/main/java/com/tc/objectserver/persistence/EntityPersistor.java

```diff
@@ -161,7 +161,6 @@ public synchronized void entityCreateFailed(EntityID eid, ClientID clientID, lon
   public synchronized void entityCreated(ClientID clientID, long transactionID, long oldestTransactionOnClient, EntityID id, long version, long consumerID, boolean canDelete, byte[] configuration) {
     LOGGER.debug("entityCreated " + clientID + " " + transactionID + " " + id + " " + version);
     Assert.assertTrue(canDelete);
-    Assert.assertFalse(clientID.isNull());
```

Comment:
this should be reverted

Reply:
See ProcessTransactionHandler.java:923.  That, I think, needs to check isPermanentEntity again

Reply:
done